tests/Doctrine/Tests/ORM/Query/QueryTest.php

```diff
@@ -242,4 +242,24 @@ public function testSetHydrationCacheProfileNull()
         $query->setHydrationCacheProfile(null);
         $this->assertNull($query->getHydrationCacheProfile());
     }
+
+    /**
+     * @expectedException        Doctrine\ORM\Query\QueryException
+     * @exptectedExcepionMessage Subquery
+     */
+    public function testSelectJoinSubquery()
+    {
+        $query = $this->_em->createQuery("select u from Doctrine\Tests\Models\CMS\CmsUser u JOIN (SELECT )");
+        $query->getResult();
+    }
+
+    /**
+     * @expectedException        Doctrine\ORM\Query\QueryException
```

Comment:
Please use `$this->expectException(QueryException::class)` and `$this->exceptExceptionMessage('Subquery')` directly before the code that is supposed to trigger the failure (I suppose it's `$query->getResult()` here?